packages/core/src/mls/MLSService/MLSService.ts

```diff
@@ -237,4 +237,8 @@ export class MLSService {
   public async wipeConversation(conversationId: ConversationId): Promise<void> {
     return this.coreCryptoClient.wipeConversation(conversationId);
   }
+
+  public async proteusCryptoboxMigrate(storeName: string) {
+    return this.coreCryptoClient.proteusCryptoboxMigrate(storeName);
```

Comment:
this will probably live in some other place (waiting for proteus client to be initialised)